packages/common-css/focus-css/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@nl-design-system-candidate/focus-css",
```

Comment:
Ik was er vanuit gegaan dat dit onder common tokens zou vallen en daarmee iets zou zijn voor volgend jaar.

Reply:
Zie ook https://github.com/nl-design-system/candidate/pull/323#discussion_r1872315490

Reply:
Goeie, heb daar een comment achtergelaten. Heb het voor nu losgeknipt in een draft pr en dan geen focus stijl toegevoegd for now (ofwel gekeken naar Amsterdam 🙈)